plugins/common/compiler/type/input.ts

```diff
@@ -0,0 +1,116 @@
+export interface CompilationInput {
```

Comment:
Does solidity team know that this exist? (and the output one)
They will appreciate it and might find some other use of it

Reply:
Yes, I told them twice. They say I would have to switch solcjs to typescript, but that this is not the priority.